### PR TITLE
Iar 34/tomas/create a new subreddit be

### PR DESCRIPTION
**Pull request by TomasBenes**

### CODE REVIEW

File: src/controllers/SubredditsController.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+@Controller('/')
+export class SubredditsController {
+	constructor(private subredditsService: SubredditsService) {}
+
+	@Post('/subreddits/create')
+	// eslint-disable-next-line max-len
+	public async createSubreddit(@BodyParams() subreddits: SubredditsCreation): Promise<SubredditsCreation> {
+		return await this.subredditsService.create(subreddits);
+	}
```
Comment (Karhalll) [inline src/controllers/SubredditsController.ts]: Is the eslint-disable necessary? I think few "Enter's" will do.

**Review by Karhalll — APPROVED** [review]
Execpt eslint-disable comment that can be resolved with few enters i think it's a clean and simple solution :)

File: src/controllers/SubredditsController.ts (hunk 1 above)
Comment (TomasBenes) [reply src/controllers/SubredditsController.ts]: Corrected with enter. Eslint activated.

File: src/controllers/SubredditsController.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { SubredditsService } from '../services/SubredditsService';
+
+
+@Controller('/')
+export class SubredditsController {
+	constructor(private subredditsService: SubredditsService) {}
+
+	@Post('/subreddits/create')
+	public async createSubreddit(@BodyParams() 
+	subreddits: SubredditsCreation): Promise<SubredditsCreation> {
```
Comment (SimProch) [inline src/controllers/SubredditsController.ts]: This indendation is quite weird and it's hard to read. see if you can update it so it is more readable

**Review by SimProch — CHANGES_REQUESTED** [review]

File: src/controllers/SubredditsController.ts (hunk 2 above)
Comment (TomasBenes) [reply src/controllers/SubredditsController.ts]: Indentation corrected

**Review by SimProch — APPROVED** [review]

**Pull request merged**